make css is broken

```
DEPRECATION WARNING on line 495, column 1 of /home/ale/Code/js/Patterns/src/pat/auto-suggest/_auto-suggest.scss:
The subject selector operator "!" is deprecated and will be removed in a future release.
This operator has been replaced by ":has()" in the CSS spec.
For example: &.undecided

Error: File to import not found or unreadable: ../../bower_components/pikaday/scss/pikaday.scss.
       Load paths:
         /home/ale/Code/js/Patterns/style
         /home/ale/Code/js/Patterns/_sass
         /home/ale/Code/js/Patterns
        on line 2 of /home/ale/Code/js/Patterns/src/pat/date-picker/_date-picker.scss
        from line 26 of _sass/_patterns.scss
  Use --trace for backtrace.
Makefile:111: recipe for target 'css' failed
make: *** [css] Error 65
****
```

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "moment-timezone": "0.5.13",
     "parsleyjs": "2.7.2",
     "photoswipe": "4.1.0",
-    "pikaday": "1.6.1",
+    "pikaday": "^1.6.1",
     "prefixfree": "",
     "select2": "3.4.5",
     "showdown": "1.7.2",

--- a/src/pat/date-picker/_date-picker.scss
+++ b/src/pat/date-picker/_date-picker.scss
@@ -1,2 +1,2 @@
 @charset "UTF-8";
-@import "../../bower_components/pikaday/scss/pikaday.scss";
+@import "../../../node_modules/pikaday/scss/pikaday.scss";

--- a/style/patterns.css
+++ b/style/patterns.css
@@ -141,7 +141,7 @@ button.pat-button[disabled="disabled"], button.pat-button:disabled {
   display: inline-block;
   overflow: hidden;
   border-radius: 50%;
-  border: 0.05556em solid rgba(0, 0, 0, 0.2);
+  border: 0.055556em solid rgba(0, 0, 0, 0.2);
   height: 2em;
   width: 2em; }
   .pat-avatar img {
@@ -315,7 +315,7 @@ form {
     /*Next 2 lines is a bizarre patch to make misbehaving FF hide the original dropdown button*/
     text-indent: 0.01px;
     text-overflow: '';
-    border-right: 0.35em solid transparent;
+    border-right: 0.35em solid rgba(0, 0, 0, 0);
     padding-right: 1em;
     background-repeat: no-repeat;
     background-position: right center;
@@ -714,7 +714,7 @@ Version: 3.4.3 Timestamp: Tue Sep 17 06:47:14 PDT 2013
   position: relative;
   overflow-x: hidden;
   overflow-y: auto;
-  -webkit-tap-highlight-color: transparent; }
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0); }
   .select2-results ul.select2-result-sub {
     margin: 0;
     padding-left: 0; }
@@ -3073,7 +3073,7 @@ label.focus {
   -webkit-user-select: none;
   -moz-user-select: none;
   user-select: none;
-  -webkit-tap-highlight-color: transparent;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
   -webkit-touch-callout: none; }
 
 .pswp__zoom-wrap {
@@ -3356,7 +3356,7 @@ a.pswp__share--facebook:before {
   position: absolute;
   top: -12px;
   right: 15px;
-  border: 6px solid transparent;
+  border: 6px solid rgba(0, 0, 0, 0);
   border-bottom-color: #FFF;
   -webkit-pointer-events: none;
   -moz-pointer-events: none;

